### DISCUSSION TITLE
Remover opção de Excluir ao usar o Po Table com selectable

### DISCUSSION THREAD

**Discussion by gtrabach** [po-ui/po-angular#1783] [Tire suas dúvidas] [answered]
Boa tarde!

Ao utilizar a versão mais recente do POUI, no componente Po Table com seleção de linhas, está sendo exibido um botão Excluir e está permitindo excluir linhas da tabela.

Como posso estar removendo essa opção? O usuário não poderá excluir linhas da tabela.

Anteriormente (versão 15.12.1) essa opção não era exibida. Na documentação, exemplo PO Table - Airfare (https://po-ui.io/documentation/po-table), é possível simular essa situação.

![botao_exclusao_tabela](https://github.com/po-ui/po-angular/assets/46979335/81765cbe-2386-4509-a026-6c37f82ad17b)

**Comment by anliben** [comment, marked as answer]
Boa Tarde, @gtrabach, tudo bem?

Utilize a propriedade `p-hide-batch-actions="boolean"`, pode ler sobre esta propriedade em nossa documentação.

https://po-ui.io/documentation/po-table

Attr.
